### PR TITLE
Sample data expanded with images and headings h1

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #124 .

### Testing Instructions
Apply sample data in a fresh installation of cassiopeia 

### Expected result
All pages have a heading h1

### Actual result
The Homepage has H1 but other pages start with h2

### Note
show_page_heading is a param in the menuItem. This means: Articles which are shown with direct link and not from the menu still have header tag 2.

### CODE REVIEW

**Review by drmenzelit — APPROVED** [review]
I have tested this PR succesfully

**Comment by richard67** [comment]
There are 2 things which should be checked and if possible improved:

1. At page "Home / About" we have a h1 page heading and a h2 article heading with the same text. That looks silly.



Possible solution: Switch off article title display in the menu item's options (not in the article options because we still might want to see the article's title in blog views) so only the h1 page heading is shown.

I haven't seen this problem on other pages.

2. In the single article views below "Home / Sample Layouts / Articles", the article title is shown as h2 heading, and there is no h1 heading. Example for the first of the 3 articles:



Possible solution same as above, hide (article) title and show page heading in menu item options.

File: plugins/sampledata/blog/blog.php [hunk 1]
```diff
@@ -323,13 +326,15 @@ public function onAjaxSampledataApplyStep1()
...
 			array(
 				'catid'    => $catIds[2],
-				'featured' => 0,
-				'ordering' => 0,
+				'featured'   => 0,
+				'ordering'   => 0,
 				'images'   => array(
-					'image_intro'            =>  'images/banners/banner.jpg',
+					'show_page_heading'      => '1',
+					'image_intro'            =>  'images/sampledata/cassiopeia/blue-rocket-400.jpg',
```
Comment (richard67) [inline plugins/sampledata/blog/blog.php]:
```suggestion
					'image_intro'            => 'images/sampledata/cassiopeia/blue-rocket-400.jpg',
```

File: plugins/sampledata/blog/blog.php [hunk 2]
```diff
@@ -344,7 +349,8 @@ public function onAjaxSampledataApplyStep1()
 				'featured' => 0,
 				'ordering' => 0,
 				'images'   => array(
-					'image_intro'            =>  'images/banners/banner.jpg',
+					'show_page_heading'      => '1',
+					'image_intro'            =>  'images/sampledata/cassiopeia/rocket-400.jpg',
```
Comment (richard67) [inline plugins/sampledata/blog/blog.php]:
```suggestion
					'image_intro'            => 'images/sampledata/cassiopeia/rocket-400.jpg',
```

File: plugins/sampledata/blog/blog.php [hunk 3]
```diff
@@ -359,7 +365,8 @@ public function onAjaxSampledataApplyStep1()
 				'featured' => 0,
 				'ordering' => 0,
 				'images'   => array(
-					'image_intro'            =>  'images/banners/banner.jpg',
+					'show_page_heading'      => '1',
+					'image_intro'            =>  'images/sampledata/cassiopeia/blue-rocket-400.jpg',
```
Comment (richard67) [inline plugins/sampledata/blog/blog.php]:
```suggestion
					'image_intro'            => 'images/sampledata/cassiopeia/blue-rocket-400.jpg',
```

**Comment by chmst** [comment]
@richard67 I fixed the h1 issue for headings in single articles.

**Comment by richard67** [comment]
> @richard67 I fixed the h1 issue for headings in single articles.

@chmst I can't confirm that. I still have both issues I described in my [comment above](https://github.com/joomla/cassiopeia/pull/147#issuecomment-700069853) .

**Comment by richard67** [comment]
> > @richard67 I fixed the h1 issue for headings in single articles.
> 
> @chmst I can't confirm that. I still have both issues I described in my [comment above](https://github.com/joomla/cassiopeia/pull/147#issuecomment-700069853) .

@chmst Have you checked my previous comment?

**Comment by chmst** [comment]
I am not quite sure abot these changes. Please do not merge

**Comment by richard67** [comment]
@chmst If you wanna be sure it's not merged, go back to draft status.

**Comment by richard67** [comment]
@chmst Meanwhile this PR will have merge conflicts with the development branch as soon as you set it to "Ready for review". Is this PR here still relevant? Or has it become obsolete meanwhile?

**Comment by chmst** [comment]
As it is mainly for switches in params and as we have no design decisions, I can close it. Thank you for remembering me

**Pull request closed without merging**